From 6e2b15013658ad631e482dbbbd50cc79dec02a66 Mon Sep 17 00:00:00 2001
From: Manoj K M <manojkm24dev@gmail.com>
Date: Sat, 25 Jul 2026 10:06:58 +0000
Subject: [PATCH 07551/15333] Documentation: fix grammar in description of
 nilfs2 recovery code

Include the word 'at' to make the sentence grammatically correct
inside the description for nilfs2 recovery code. This improves
documentation without changing the original meaning.

Signed-off-by: Manoj K M <manojkm24dev@gmail.com>
Acked-by: Ryusuke Konishi <konishi.ryusuke@gmail.com>
Signed-off-by: Viacheslav Dubeyko <slava@dubeyko.com>
---
 Documentation/filesystems/nilfs2.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/filesystems/nilfs2.rst b/Documentation/filesystems/nilfs2.rst
index e3a5c8977f2c..a0124d22a237 100644
--- a/Documentation/filesystems/nilfs2.rst
+++ b/Documentation/filesystems/nilfs2.rst
@@ -252,7 +252,7 @@ The following figure shows a typical organization of the logs::
 To stride over segment boundaries, this sequence of files may be split
 into multiple logs.  The sequence of logs that should be treated as
 logically one log, is delimited with flags marked in the segment
-summary.  The recovery code of nilfs2 looks this boundary information
+summary.  The recovery code of nilfs2 looks at this boundary information
 to ensure atomicity of updates.
 
 The super root block is inserted for every checkpoints.  It includes
-- 
2.55.0

